internal/tasks/tasks.go

```diff
@@ -1196,7 +1190,7 @@ func taskGetSystemUptime() string {
 func taskGetStorageDevices() string {
 	fmt.Println("Executing taskGetStorageDevices")
 
-	devices := storage.GetDevices(diagnostics.GetReleaseVersion())
```

Comment:
What is the reasoning to not pass the version (flavour) here? Since for now it is a bit fuzzy on how we wanna deal with reporting storage devices and the whole logic of mounting them and making them available for apps, we can go with not passing the version and then deal with the storage feature part as its own thing :)